haystack/backends/elasticsearch_backend.py

```diff
@@ -186,8 +186,11 @@ def update(self, index, iterable, commit=True):
 
         bulk_index(self.conn, prepped_docs, index=self.index_name, doc_type='modelresult')
 
-        if commit:
-            self.conn.indices.refresh(index=self.index_name)
+        # We are explicitly eliminating the call to self.conn.indices.refresh().
```

Comment:
do we need to be concerned about events that happen within that 1 minute period - e.g. multiple saves of the same data?  Or, would anything in SM write data and then immediately read it?  Is that a problem?